expressEngine/user/service/user.js

```diff
@@ -49,6 +49,16 @@ class Service extends BaseService {
 
     return user;
   }
+
+  delete = async (id) => {
+
+    const isExist = await DBService.count(User, { userId: id });
```

Comment:
@adilsuccessive Prefer using countDocuments, but here we can use findOne as well to check if user exists.

Reply:
Done with the change, please check